src/store/authentication/authentication.thunk.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { url as baseUrl } from '../../utils/constants';
+
+export const Login = createAsyncThunk('login', async (data, { rejectWithValue }) => {

```

Comment:
```suggestion
export const login = createAsyncThunk('login', async (data, { rejectWithValue }) => {

```
Đặt tên in hoa đầu chỉ dành cho các class hay component thôi nhé